### ISSUE TITLE
Fails to run on nodejs 6.10.3

### ISSUE THREAD

**Issue by michaelkruglos** [singapore/renovate#352]
## Problem
Current version of renovate fails to run on nodejs 6.10.3.

### Error message

```javascript
/usr/local/lib/node_modules/renovate/lib/api/github.js:42
async function getInstallations(appToken) {
      ^^^^^^^^
SyntaxError: Unexpected token function
    at createScript (vm.js:56:10)
    at Object.runInThisContext (vm.js:97:10)
    at Module._compile (module.js:542:28)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/usr/local/lib/node_modules/renovate/dist/workers/repository.js:266:19)
```

### Docker
docker run --rm -it node:6.10.3-slim "/bin/sh" "-c" "npm install -g renovate && renovate --version"

### Non-Docker

- Install nodejs 6.10.3
- Install renovate: `npm install -g renovate`
- Try to run it: `renovate --version`

**Comment by rarkins** [comment]
Thanks for the bug report. It's meant to be transpiled but apparently something is broken. Pretty sure it is meant to be running out of dist/ and not lib/. 

I'll take a look later today.

**Comment by rarkins** [comment]
I think I messed up in a recent refactor and made some internal requires specific to lib: https://github.com/singapore/renovate/blob/master/lib/helpers/platform.js

Step back 1-2 minor releases and it might be fine, in case it's urgent for you. Otherwise I'll be fixing once back at my computer later today.

**Comment by rarkins** [comment]
Hopefully fixed by #353 but I'll confirm and publish when at computer later.

**Comment by michaelkruglos** [comment]
Thanks for the quick response 👍

**Comment by rarkins** [comment]
Fixed in renovate@8.40.1. Sorry for the inconvenience and thanks for the bug report
